src/ValidationCheck.js

```diff
@@ -0,0 +1,53 @@
+class ValidationCheck {
+    static consistsOfPositiveNumber(str) {
+        const NUMBER_REGEXP = /^[1-9]+$/;
+        if (!NUMBER_REGEXP.test(str)) {
+          return false;
+        }
+        return true;
+    }
+
+    static isThreeDigit(str) {
+        if (!ValidationCheck.consistsOfPositiveNumber(str) || str.length !== 3) {
+          return false;
+        } 
+        return true;
+    }
+
+    static isAllDifferent(str) {
+        const setToCompare = new Set(str);
+        if (str.length !== setToCompare.size) {
+          return false;
+        }
+        return true;
```

Comment:
여기도 @kangju2000 님이 해주신 조언과 겹치는데요! 이렇게 해주시면 좀더 보기 좋을것 같아요:)
```suggestion
        return str.length === setToCompare.size
```

Reply:
정말 고쳐야할 부분이 많네요.. 오늘도 반성합니다. 좋은 리뷰 감사합니다!!